.github/workflows/deploy-mkdocs.yml

```diff
@@ -20,14 +20,12 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v5
-        with:
-          python-version: ${{ matrix.python-version }}
+      - uses: astral-sh/setup-uv@v5
 
       - name: Install dependencies
         run: python -m pip install -e .[pgstac,pc,test,docs]
-
+      - name: Sync
+        run: uv sync --all-extras
```

Comment:
We can skip this step and just use `uv run` in the `mike deploy` call, see https://github.com/stac-utils/rustac-py/blob/main/.github/workflows/docs.yaml#L38 for a ref. For this repo, it would be `uv run --group docs mike deploy ...`

```suggestion
```